### ISSUE TITLE
[Suggestion] Adding Overworld Silicon Ore to Void Ore Miner

### ISSUE THREAD

**Issue by pjc21** [DarkPacks/SevTech-Ages#2684]
Seems most other overworld ores can be obtained from the void ore miner.
Suggest to possibly add the overworld silicon ore to the void ore miner, unless this has been left out intentionally.
- galacticraftcore:basic_block_core:8

**Comment by GearsOfClockwork** [comment]
I agree with this as well. With the mystical agriculture recipe not working, silicon is hard to obtain in large quantities, unless you plan on adding back the sand-to-silicon recipe.
